editor/plugins/spatial_editor_plugin.cpp

```diff
@@ -1422,7 +1422,7 @@ void SpatialEditorViewport::_sinput(const Ref<InputEvent> &p_event) {
 
 						Vector3 motion_mask;
 						Plane plane;
-						bool plane_mv = false;
+						bool plane_mv = false;	
```

Comment:
Looks like a bad formating here

Reply:
Use `git commit --amend` command to fix it (and then git push --force origin <branch (master in your case> - http://docs.godotengine.org/en/latest/community/contributing/pr_workflow.html , https://git-scm.com/book/en/v2/Git-Basics-Undoing-Things

Reply:
Its generally recommended to create a separate branches for each PR you've created

Reply:
From now on I will take care of these things.